src/components/list/list.ts

```diff
@@ -48,13 +57,20 @@ export class MdListItem implements AfterContentInit {
     this._renderer.setElementClass(this._element.nativeElement, 'md-list-avatar', avatar != null);
   }
 
-  constructor(private _renderer: Renderer, private _element: ElementRef) {}
+  constructor(private _renderer: Renderer, private _element: ElementRef) {
+    this._hasLink = _element.nativeElement.tagName.toLowerCase() == 'a';
```

Comment:
Finally got around to updating this for the gulp/NgModule changes. It now only adds ripples to items in `md-nav-list` components, not `md-list`.